### PR TITLE
Update README with synth data + env options

### LINKED ISSUE #33: Installation doesn't work with python>=3.10
Installing `grace` via `pip install -e .` doesn't work with `python>=3.10` version. 
Need to downgrade python to 3.9.

### LINKED ISSUE #43: Napari annotator installation via `conda-forge`
The napari annotator works after:

```sh
pip uninstall napari
conda install -c conda-forge napari
```
or with the downgraded version of `pydantic==1.10.2`

https://github.com/alan-turing-institute/grace/pull/20#discussion_r1158762176

### LINKED ISSUE #135: Create an environment.yaml which contains napari
We should ideally have 2 environments: one that is used by people who don't need to open images + annotations in napari (napari-free, ✅ ) and one that installs napari to allow annotation + enables opening of the `examples` scripts 🛑 .

### LINKED ISSUE #153: Specify difference between two conda environments

### LINKED ISSUE #154: Fix the `.yml` vs `.yaml` typo in the new napari environment

### LINKED ISSUE #155: Motivate the user to check out the synthetic dataset creation option

### PR DESCRIPTION
**Pull request by KristinaUlicna**
- [x] Resolves #153
- [x] Resolves #154
- [x] Resolves #155

Solving some older issues which weren't tagged in previous PR documentation:

- [x] Resolves #135
- [x] Resolves #43
- [x] Resolves #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,27 +57,34 @@ The `grace` workflow consists of the following steps:
 
 `grace` has been tested with Python 3.8+ on OS X.
 
-For local development, clone the repo and install in editable mode:
+For local development, clone the repo and install in editable mode following these guidelines:
+
+*Note:* Choose which conda environment you'd like to use:
+* If you need to annotate your data (images) in *napari*, we recommend the `grace-env-with-napari` -> `environment-with-napari.yaml`
+* If you rely on previously annotated data and do not require *napari*, we recommend the `grace-env-napari-free` -> `environment-napari-free.yaml`
```
Comment (mooniean) [inline README.md:64]: Having napari in both names might be confusing (or confused with a napari "free" version as opposed to paid?). Maybe something like environment-without-annotation or something along those lines

File: README.md [hunk 2]
```diff
@@ -92,27 +99,10 @@ conda install -c conda-forge pre_commit
...
-#
-#     $ conda deactivate
-
-conda activate grace-annotation
-
-```
-
 ---
 
+If you currently do not have any data to test / implement GRACE on, have a look at the option of **simulating a synthetic dataset** as described in [this](./grace/simulator/README.md) README. An assessible link to some pre-annotated simulated images is coming soon! 🚧
```
Comment (mooniean) [inline README.md:104]: accessible?

**Review by mooniean — COMMENTED** [review]
Just minor changes, but I like the new format in the readme!

File: README.md (hunk 2 above)
Comment (KristinaUlicna) [reply README.md:104]: The one I created in on the Turing OneDrive, we need to agree how to share the data (usually not the best practice to have data in the repo, as they are quite large files).

File: README.md (hunk 1 above)
Comment (mooniean) [reply README.md:64]: did anything change related to this comment?

**Review by chris-soelistyo — APPROVED** [review]
Looks great!

**Pull request merged**